docs/guides/js-waku/quick-start.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Quick Start
+---
+
+This guide provides quick steps to start using the `js-waku` SDK by setting up a Waku node and sending messages using the [Relay protocol](/overview/concepts/protocols#relay). Please refer to the [installation guide](/guides/js-waku/#installation) for steps on adding `js-waku` to your project.
+
+## Create a Relay Node
+
+Use the `createRelayNode()` function to create a relay node and interact with the Waku Network:
+
+```js
+import { createRelayNode } from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({ defaultBootstrap: true });
+await node.start();
+
+// Use the stop() function to stop a running node
+// await node.stop();
+```
+
+:::info
+The `defaultBootstrap` option bootstraps your node using [pre-defined Waku nodes](/overview/concepts/static-peers).
```

Comment:
Default bootstrap use dns discovery using the wakuv2.prod domain name

https://github.com/waku-org/js-waku/blob/2ceac3bdf31a7553dae55d75949a26aa0e3c0f67/packages/sdk/src/create.ts#L174